poetry.lock

```diff
@@ -7,7 +7,7 @@ optional = false
 python-versions = ">=3.6"
```

Comment:
I don't believe you should change `poetry.lock` if there are no changes to `pyproject.toml` but this might be an error in my setup of poetry :thinking:. For now I'd rather you not merge these kind of changes :thinking: but we should double check that together